Sources/SwiftProtobuf/JSONScanner.swift

```diff
@@ -79,41 +79,143 @@ private func fromHexDigit(_ c: UnicodeScalar) -> UInt32? {
   }
 }
 
+let base64Values: [Int] = [
```

Comment:
It's the regular one only.

I went back and looked more carefully; although the C++ libraries support a lot of base64 variants, the conformance test and Java versions make clear that only the regular base64 encoding should be supported.

I've updated the comments here to clarify.